blockchain/genesis/genesis.go

```diff
@@ -39,19 +39,20 @@ func init() {
 func defaultConfig() Genesis {
 	return Genesis{
 		Blockchain: Blockchain{
-			Timestamp:              1546329600,
-			BlockGasLimit:          20000000,
-			ActionGasLimit:         5000000,
-			BlockInterval:          10 * time.Second,
-			NumSubEpochs:           2,
-			NumDelegates:           24,
-			NumCandidateDelegates:  36,
-			TimeBasedRotation:      false,
-			PacificBlockHeight:     432001,
-			AleutianBlockHeight:    864001,
-			BeringBlockHeight:      1512001,
-			CookBlockHeight:        1641601,
-			DardanellesBlockHeight: 1855201,
+			Timestamp:               1546329600,
+			BlockGasLimit:           20000000,
+			ActionGasLimit:          5000000,
+			BlockInterval:           10 * time.Second,
+			NumSubEpochs:            2,
+			DardanellesNumSubEpochs: 30,
+			NumDelegates:            24,
+			NumCandidateDelegates:   36,
+			TimeBasedRotation:       false,
+			PacificBlockHeight:      432001,
+			AleutianBlockHeight:     864001,
+			BeringBlockHeight:       1512001,
+			CookBlockHeight:         1641601,
+			DardanellesBlockHeight:  1816201,
```

Comment:
this block translates to Tues 11/19 3:49pm

can we change it to around 9~10am? in case sth happens, we have the remaining day to check

Reply:
after discuss, 3:49pm caters to delegates in Asian time zone